src/__tests__/realmApplicationRoles.test.ts

```diff
@@ -75,7 +75,7 @@ describe('Realm Application Roles', () => {
       realmName,
       applicationId,
       {
-        name: 'Chef',
+        name: 'Head Chef',
```

Comment:
switched all tests to include space character which caused tests to fail until we added the encoding to the route